options.go

```diff
@@ -311,11 +311,11 @@ type RawCommand struct {
 	cmds []string
 }
 
-func NewRawCommand(cmds []string) RawCommand {
+func NewRawCommand(cmds []string, opts ...tcexec.ProcessOption) RawCommand {
```

Comment:
bug: this is a breaking change as users of this exported type would receive compilation errors. Have you considered adding a functional option, or adding a new function?

Reply:
If there was a compilation error wouldn't the linter fail and the test not start?
I think it is fine because it is a spreaded arguments, making it optional. If nothing is provided then `opts` is an empty slice of `tcexec.ProcessOption`.

Reply:
Also, both cases are covered by a test in `options_test.go` line 149 and 178.

Reply:
Nonetheless, happy to consider other ways to achieve that!

Reply:
@mdelapenya as I was currious about that edge case I tried a reproduction case in that [playground](https://go.dev/play/p/kuGQI7isstS). I didn't manage to make it fail, did I miss something?